packages/components/src/components/input/input.tsx

```diff
@@ -103,69 +103,78 @@ const Input = React.forwardRef<HTMLInputElement & HTMLTextAreaElement, TInputPro
-                            onChange={props.onChange}
-                            onPaste={props.onPaste}
-                            disabled={disabled}
-                            data-lpignore={props.type === 'password' ? undefined : true}
-                            id={input_id}
-                            aria-label={label as string}
-                            maxLength={maxLength}
-                        />
-                    )}
-                    {trailing_icon &&
-                        React.cloneElement(trailing_icon, {
-                            className: classNames('dc-input__trailing-icon', trailing_icon.props.className),
-                        })}
-                    {label && (
-                        <label className={classNames('dc-input__label', label_className)} htmlFor={props.id}>
-                            {label}
-                        </label>
-                    )}
-                    {!has_footer && (
-                        <React.Fragment>
-                            {error && <Field className={classNameError} message={error} type='error' />}
-                            {warn && <Field className={classNameWarn} message={warn} type='warn' />}
-                            {!error && hint && !is_relative_hint && (
-                                <div className='dc-input__hint'>
-                                    <Text
-                                        as='p'
-                                        color='less-prominent'
-                                        size='xxs'
-                                        className={classNames(classNameHint)}
-                                    >
-                                        {hint}
-                                    </Text>
-                                </div>
-                            )}
-                        </React.Fragment>
-                    )}
+                        {label && (
+                            <label className={classNames('dc-input__label', label_className)} htmlFor={props.id}>
+                                {label}
+                            </label>
+                        )}
+                    </div>
+                    <div>
+                        {!has_footer && (
+                            <React.Fragment>
+                                {error && <Field className={classNameError} message={error} type='error' />}
+                                {warn && <Field className={classNameWarn} message={warn} type='warn' />}
+                                {!error && hint && !is_relative_hint && (
+                                    <div className='dc-input__hint'>
+                                        <Text
+                                            as='p'
+                                            color='less-prominent'
+                                            size='xxs'
+                                            className={classNames(classNameHint)}
```

Comment:
Can we remove `classNames` here?

```suggestion
                                            className={classNameHint}
```